pages/11.miscellaneous/03.plurals/docs.en.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Plural forms
+taxonomy:
+    category:
+        - docs
+visible: true
+---
+
+* [Introduction](#intro)
+* [What do plural forms mean?](#plurals)
+* [Where can you learn about the number of plural forms?](#where_to_learn)
```

Comment:
It's better to use dashes `-` instead of underscores `_` in slugs. E.g. "where-to-learn" and not "where_to_learn".